app/assets/stylesheets/catarse_bootstrap/_extra.sass

```diff
@@ -405,3 +405,6 @@ img, iframe
   100% 
     +transform(translateY(-50%) scale(0))
     opacity: 0
+    
+.w-hidden
+  display: none !important
```

Comment:
Are you sure this won't break any JS that hides and shows elements dynamically ?